account_commission/wizards/commission_make_settle.py

```diff
@@ -14,16 +14,19 @@ class CommissionMakeSettle(models.TransientModel):
         ondelete={"sale_invoice": "cascade"},
     )
 
+    def _get_account_settle_domain(self, agent, date_to_agent):
```

Comment:
This is outside the scope of the PR. I'm not against it. Just that is should be done apart.